apps/data-norge/public/content/docs/finding-data/resource-service/resource-service.nb.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Ressurstjeneste
+description: Ressurstjenesten gir deg direkte tilgang til ressurser fra data.norge.no gjennom REST API.
+---
+
+# Ressurstjeneste
```

Comment:
Jeg lurer på om det blir litt for spesifikt å ha en egen side om ressurstjenesten her? Kanskje denne siden bare kan hete "API"? Så kan vi bruke denne siden til å opplyse om at vi har flere API-er i tillegg til de andre måtene å manuelt finne data på - blant annet en ressurstjeneste.

Og så bare lenke videre til `nb/technical/api` herfra? Så slipper vi å gjenta så mye av informasjonen man finner under [teknisk dokumentasjon](https://staging.fellesdatakatalog.digdir.no/nb/technical/api) her?

<img width="1250" height="542" alt="Image" src="https://github.com/user-attachments/assets/84fb2cba-e6bc-453d-b7ca-cc2b402e3c61" />
<img width="1253" height="545" alt="Image" src="https://github.com/user-attachments/assets/c335a1fa-d5c0-46ed-b4c8-d140849b2cfb" />